packages/moonstone/IncrementSlider/IncrementSlider.js

```diff
@@ -96,7 +95,7 @@ const IncrementSliderBase = kind({
 		'data-webos-voice-group-label': PropTypes.string,
 
 		/**
-		 * When `true`, the knob displays selected and can be moved using 5-way controls.
+		 * Displays the knob selected and can be moved using 5-way controls.
```

Comment:
Should this be "knob as selected" or "knob in selected state". Reading this makes me feel like there might be multiple knobs.